silo/forms.py

```diff
@@ -57,6 +59,21 @@ class Meta:
         fields = ['id', 'name', 'description', 'tags', 'shared',
                   'share_with_organization', 'owner', 'workflowlevel1']
 
+    def is_valid(self):
```

Comment:
You shouldn't overwrite this function. You should do something like we did for the field `org` in this form https://github.com/Humanitec/ActivityAPI/blob/master/web/forms.py